dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -99,3 +99,11 @@ lease.renewal.threshold.desc=The percentage of the duration of the recovery log
 enable.log.retries=Enable retries on database transaction logs
 enable.log.retries.desc=When recovery logs are stored in an RDBMS table, this property allows SQL operations that fail to be retried.
 
+shutdown.on.log.failure=Shutdown server on transaction log failure
+shutdown.on.log.failure.desc=Specifies if an application server that is part of a recoveryGroup will be terminated by the transaction manager if a non-recoverable error occurs accessing its own transaction service logs.
+
+log.retry.interval=Interval between tran log retries on failure
```

Comment:
Is "tran" used for a log name? If not, use "transaction"

Reply:
My initial Personal Build failed because there is a 50 character limit on label strings. I shortened the words so as not to break the limit but to try to keep the meaning of the label. I'm very happy to take suggestions for improvements that stay in that 50 character limit. Thanks